CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# Contributing to Heroic
+
+Thanks for taking the time to contribute! 
+
+The following is a set of guidelines for contributing to Heroic. 
+
+## Ground Rules
+
+* Be sure to add a comment to your PR explaining the reason for the change. Please include some context behind a bug or issue you ran in to, or your use case and why you need a particular feature added. 
+* Include tests for any large changes. Read here to learn how to run the [tests](https://github.com/spotify/heroic#testing).
```

Comment:
We should have a bullet saying to open an issue for discussion before submitting any major PRs.

And in addition to tests, documentation should be updated in the PRs.